### PR TITLE
Missing Mint UI logo fix

### LINKED ISSUE #365: [BUG] Mint UI logo missing
**Describe the bug**
Mint UI on the Vue Libraries is missing a logo: https://vue.framework.dev/categories/libraries

Screen Shot 2022-10-29 at 12 52 44 PM

Use the logo from their website: https://mint-ui.github.io/#!/en

### PR DESCRIPTION
**Pull request by suchitd11**
## Type of change

- [ ] Content addition
- [x] Bug fix
- [ ] Behavior change

## Summary of change
Added link to the logo of Mint UI in the libraries file.

## Checklist

- [x] This fix resolves #365 
- [x] The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
- [x] I have verified the fix works and introduces no further errors

### CODE REVIEW

**Comment by suchitd11** [comment]
@dustinsgoodman please accept my pull request!

File: packages/site/src/data/vue/libraries.ts [hunk 1]
```diff
@@ -261,7 +261,7 @@ export const libraries: Library<typeof libraryTags[number]>[] = [
 		npmPackage: "vuestic-ui",
 		href: "https://vuestic.dev/",
 		description: "Vue 3.0 UI Framework",
-		image: "",
+		image: "https://raw.githubusercontent.com/epicmaxco/vuestic-ui/develop/packages/ui/src/assets/vuestic-ui-logo.png",
```
Comment (jdwilkin4) [inline packages/site/src/data/vue/libraries.ts:264]: You can remove this change since it is already being addressed in your other PR.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
I tested out the Mint UI logo and it looks good to me.
You just need to remove the changes you made for the vuestic image link since you are already working on that in another PR.
Then we can merge this PR 👍

File: packages/site/src/data/vue/libraries.ts (hunk 1 above)
Comment (suchitd11) [reply packages/site/src/data/vue/libraries.ts]: done!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by suchitd11** [comment]
@jdwilkin4 Thank you!! It was great experience under your guidance😄

**Pull request merged**